conda-store-server/conda_store_server/conda_store.py

```diff
@@ -3,365 +3,34 @@
-            schema.BuildArtifactType.LOGS,
-            schema.BuildArtifactType.LOCKFILE,
-            schema.BuildArtifactType.YAML,
-        ],
-        help="artifacts to keep on build deletion",
-        config=True,
-    )
-
-    serialize_builds = Bool(
-        True,
-        help="DEPRICATED no longer has any effect",
-        config=True,
-    )
-
-    @default("celery_broker_url")
-    def _default_celery_broker_url(self):
-        if self.redis_url is not None:
-            return self.redis_url
-        return f"sqla+{self.database_url}"
-
-    celery_results_backend = Unicode(
-        help="backend to use for celery task results",
-        config=True,
-    )
-
-    @default("celery_results_backend")
-    def _default_celery_results_backend(self):
-        if self.redis_url is not None:
-            return self.redis_url
-        return f"db+{self.database_url}"
-
-    default_namespace = Unicode(
-        "default", help="default namespace for conda-store", config=True
-    )
-
-    filesystem_namespace = Unicode(
-        "filesystem",
-        help="namespace to use for environments picked up via `CondaStoreWorker.watch_paths` on the filesystem",
-        config=True,
-    )
-
-    default_uid = Integer(
-        None if sys.platform == "win32" else os.getuid(),
-        help="default uid to assign to built environments",
-        config=True,
-        allow_none=True,
-    )
-
-    default_gid = Integer(
-        None if sys.platform == "win32" else os.getgid(),
-        help="default gid to assign to built environments",
-        config=True,
-        allow_none=True,
-    )
-
-    default_permissions = Unicode(
-        None if sys.platform == "win32" else "775",
-        help="default file permissions to assign to built environments",
-        config=True,
-        allow_none=True,
-    )
-
-    validate_specification = Callable(
-        conda_store_validate_specification,
-        help="callable function taking conda_store, namespace, and specification as input arguments to apply for validating and modifying a given specification. If there are validation issues with the environment ValueError with message should be raised. If changed you may need to call the default function to preseve many of the trait effects e.g. `c.CondaStore.default_channels` etc",
-        config=True,
-    )
-
-    validate_action = Callable(
-        conda_store_validate_action,
-        help="callable function taking conda_store, namespace, and action. If there are issues with performing the given action raise a CondaStoreError should be raised.",
-        config=True,
-    )
-
-    post_update_environment_build_hook = Callable(
-        default_value=None,
-        help="callable function taking conda_store and `orm.Environment` object as input arguments. This function can be used to add custom behavior that will run after an environment's current build changes.",
-        config=True,
-        allow_none=True,
-    )
+class CondaStore:
```

Comment:
Would it be possible to add some documentation about _what this is_? What does this class do, how is it different from the traitlets/FastAPI `CondaStoreServer` application, etc would be very helpful here.

Really thing this seems like a celery worker manager - maybe a different name could provide a clearer idea what this class is for. Or maybe the webserver should be named differently :thinking: What do you think?